root/layout/components/sidebar/SidebarTags.js

```diff
@@ -55,46 +55,47 @@ const SidebarTags = ({
 }: SidebarTagsProps) => (
   $c.action.name === 'tags' ? null : (
     <>
-      {($c.user && aggregatedTags && userTags) ? (
-        <SidebarTagEditor
-          $c={$c}
-          aggregatedTags={aggregatedTags}
-          entity={entity}
-          more={more}
-          userTags={userTags}
-        />
-      ) : (
-        <div id="sidebar-tags">
-          <h2>{l('Genres')}</h2>
-          <div className="genre-list">
-            <p>
-              <TagList
-                entity={entity}
-                isGenreList
-                tags={aggregatedTags}
-              />
-            </p>
-          </div>
+      {($c.user && $c.user.has_confirmed_email_address &&
+        aggregatedTags && userTags) ? (
```

Comment:
Optional chaining might be nice here. (I count 4 other places in the PR, too)
```suggestion
      {($c.user?.has_confirmed_email_address &&
        aggregatedTags && userTags) ? (
```